Prace_domowe/Praca_domowa5/Jakub_Foltyn/PD5.ipynb

```diff
@@ -0,0 +1 @@
1m4YBiOHvMeCH5fh9/qxO20E/AHLSJiA3yA7PVwHX6FQFB2l6nmBklilLM9+8zgxpaOJLhVFVJyLhAqlef23/xIVc1Jn5l+PXsNtr90SkhF8xcCOvDXvecvjOqMc2PNx/ThjnFxzz1URw0QjxaIHvH5KlY0La//r28Us/GkZniwPAX/ATFDz+HMqfOXGFeOkfb82Ye0KhaLoqJn/BUyb3i345vB4Ni/Zjm7Xqde6VphLRAjBwKf6M/Cp/hiGgabl/7y32W30vqM70z7+3VLDx+v20bZvSya/ksAhd1JO4RPdphNfvhRxZWLZvT68yLyUkmUzV9PhulDj/etncyPG8Qsh8PvMTGVXjJO2fVvSonuTAu+LQqEoGDXzv8CxO+w06dSAhu3qFugLL8jwn+CuN28loYJ1IXW7w8aGBVsYveAVut7cAWeUA4fLTqd/teXDFW9Qv4111TJpGGSmZVm0W7uOHC479707jL539uDKIV3471eP8cyUR1WWq0JxhlAzf0UYdoedpp0bMmfSX+EbpZl9W6psHE9NeJCnJjwYsvnya1sz/+uFuDNCZ/MBv0HLHuGz9p7DurJx0ZbwkFVdo9dt3Qql6qlQKIqOmvkrLOlzZw9Lv77NaaNZl8hCrO2uvozGHernVBQTwnxY3PjkdSRWKRvWv8tN7bnsyqZmf2HO+F3RTv7vq8eU4VcoihFVzEURkYkvfcPkV3/AZteDJRs1Xpv5DPXb1Ml3v4A/wF/fLWb+VwtwxbjofUd3mnWJXIhESsn6vzezau46SpWJo+vN7YkvZ+12UigUhSe/DF9l/BX5cvRACqvnrie6VBStejbH4bSf6yEpFIpCUmzyDoqLn8TKZegxuPO5HoZCoTjDKJ+/QqFQlECU8VcoFIoSiDL+CoVCUQJRxl+hUChKIMr4KxQKRQlERfsoisSh3UnMmjCf1MNptLqqGe2uuUxJLCsUFyDK+CsKzeJpK3j5plEEAgZ+r585E/+kdouajPz9/3IKtysUigsD5fZRFAqf18frt47Bk+3NUfvMznCzdcVOfvvf/HM7OIVCUWSU8VcUii3LdlhW7/JkeZgz6c9zMCKFQnE6KOOvKBR2pz2i/HJB9YIVCsX5hzL+ikJRp2VNYkpHh7W7Ypz0vevKczAihUJxOijjrygUmqbx0i//Jq5MLNFxUTijnTiiHPS4tQsdB7Q918NTKBRFREX7KApN7eY1mbp/HEumryQ9OZ2mXRpStW7lcz0shUJxCijjrygSDqedTmqmr1Bc8Ci3j0KhUJRAlPFXKBSKEogy/gqFQlECUcZfoVAoSiDK+CsUCkUJ5IIo4C6EOAL8cxZOlQgcPQvnudBQ98UadV+sUffFmnNxX6pLKctZbbggjP/ZQgixPFKl+5KMui/WqPtijbov1pxv90W5fRQKhaIEooy/QqFQlECU8Q9l3LkewHmKui/WqPtijbov1pxX90X5/BUKhaIEomb+CoVCUQJRxl+hUChKICXW+AshbhBCbBBCGEKIVnm2PS2E2C6E2CKE6JmrvVewbbsQ4t9nf9RnFyHE80KI/UKI1cG/Prm2Wd6jkkJJ+y7khxBitxBiXfA7sjzYVkYI8bsQYlvwvwnnepzFjRDiMyFEkhBifa42y/sgTMYEvz9rhRAtz/Z4S6zxB9YDA4CQArRCiIbAzUAjoBfwgRBCF0LowPtAb6AhMDDY92LnHSll8+DfDIh8j87lIM8mJfi7kB9XBL8jJyZS/wbmSCnrAHOCny92Psf8PeQm0n3oDdQJ/t0FfHiWxphDiTX+UspNUsotFpv6AVOllB4p5S5gO9Am+LddSrlTSukFpgb7lkQi3aOSgvouFEw/YELw3xOA687hWM4KUso/gZQ8zZHuQz/gC2myGIgXQlQ6OyM1KbHGPx+qAHtzfd4XbIvUfrHzQPC19LNcr+4l9V6coKRff14kMEsIsUIIcVewrYKU8mDw34eACudmaOecSPfhnH+HLupKXkKI2UBFi03PSCl/OtvjOR/J7x5hvoq+hPnjfgl4G7jt7I1OcYHQUUq5XwhRHvhdCLE590YppRRClPiY8vPtPlzUxl9K2eMUdtsPXJLrc9VgG/m0X7AU9h4JIT4BpgU/5nePSgIl/fpDkFLuD/43SQjxA6Zb7LAQopKU8mDQnZF0Tgd57oh0H875d0i5fcL5GbhZCOEUQtTEXJBZCiwD6gghagohHJgLnj+fw3EWO3l8kP0xF8kh8j0qKZS470IkhBAxQoi4E/8GrsL8nvwMDA12GwqU1DftSPfhZ2BIMOqnHZCWyz10VrioZ/75IYToD7wHlAOmCyFWSyl7Sik3CCG+BjYCfuB+KWUguM8DwG+ADnwmpdxwjoZ/tnhDCNEc0+2zG7gbIL97VBKQUvpL4HchEhWAH4QQYNqTyVLKmUKIZcDXQojbMeXYbzyHYzwrCCGmAF2BRCHEPuA54HWs78MMoA9msEQWMPysj1fJOygUCkXJQ7l9FAqFogSijL9CoVCUQJTxVygUihKIMv4KhUJRAlHGX6FQKEogyvgrFApFCUQZf4VCoSiB/D8xULCxEBKe6AAAAABJRU5ErkJggg==\n","text/plain":["<Figure size 432x288 with 1 Axes>"]},"metadata":{"tags":[],"needs_background":"light"}}]},{"cell_type":"markdown","metadata":{"id":"D54wuDmwFJCs"},"source":["Podsumowując, choć być może intuicyjnie (patrząc na wykresy) 6 klastrów wydaje się być lepszym podziałe, to jednak zarówno wykres \"łokciowy\" w metodzie k-średnich i dendrogram w metodzie hierarchicznej wskazują na 3 klastry."]}]}
```

Comment:
<p>Dobrze wykonana PD.</p><ul><li>podobało mi się, że skomentowano wszystkie otrzymane wyniki</li><li>zabrakło zastosowania takiej samej miary np. silhouette aby porównać wyniki otrzymane metodą k-means i hierarchicznej (mimo, że obie wskazują na k=3 to można zauważyć, że wynik grupowania nie jest taki sam)</li></ul><p> </p><p>Merge zostanie wykonany po oddaniu PD przez inne grupy</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>